tutorials/using-cloud-vpn-with-checkpoint/index.md

```diff
@@ -0,0 +1,383 @@
+
+**Note:** Make sure you select Perfect Forward Secrecy (Phase 2). This example refers to IKEv1. You can also use IKEv2 in this scenario.
+
+![alt_text](Image_6.PNG)
+
+**Step 6**. Go to the **Advanced** tab and modify the Renegotiation Time.
+
+* **IKE for Phase 1**: 610 minutes
+* **IKE for Phase 2**: 10,800 seconds
+
+![alt_text](Image_7.PNG)
+
+**Step 7**. Configure the Access Control Rule Base and Install policy.
+
+For more information, see the R80.10 Site To Site VPN Administration [Guide](http://dl3.checkpoint.com/paid/ea/ea41387591dcba2a8d551ba39084e9e6/CP_R80.10_SitetoSiteVPN_AdminGuide.pdf?HashKey=1515459944_c0affaeb9262c888e85d660e781d604d&xtn=.pdf).
+
+# Configuring Route Based IPsec VPN Tunnel
+
+Below is a sample environment to walk you through set up of route based VPN. Make sure
+to replace the IP addresses in the sample environment with your own IP addresses.
+
+**Google Cloud Platform**
+
+|Name | Value|                             
+-----|------                                  
+|GCP(external IP)|`35.195.227.26`|
+|VPC CIDR|`10.132.0.0/20`|
+|TUN-INSIDE GCP|`169.254.0.1`|
+|GCP-ASN|`65000`|
+                        
+**Check Point**
+
+|Name | Value|
+-----|------
+|Check Point Security Gateway(external IP)|`199.203.248.181`|
+|Addresses behind Check Point Security Gateway|`10.0.0.10/24`|
+|TUN-INSIDE- CP|`169.54.0.2`|
+|CP Security Gateway ASN|`65002`|
+
+## Configuring Cloud VPN
+
+### Configuring Cloud Router
+
+The Google Cloud Router dynamically exchange routes between your Virtual Private Cloud and on-premise networks with Border Gateway Protocol (BGP). For the initial release, Cloud Router supports BGP for Cloud VPN only. Cloud Router works with both legacy networks and sub-networks.
+
+Create a new Cloud router to configure GCP for Site to Site VPN connectivity.
+
+In Google Cloud Platform Console, select **Networking** > **[Cloud Routers](https://console.cloud.google.com/interconnect/routers)** > **Create Router**.
+
+Enter the parameters as shown in the following table and click **Create**.
+
+|Parameter|Value|Description|
+|---------|-----------|-----|
+|Name|`gcp-to-cp-router`|Name of the cloud router.|
+|Description|           |Description of the cloud router.|
+|Network|`to-cp`|The GCP network the cloud router attaches to. Note: This is the network which manages route information.|
+|Region|`europe-west1`|The home region of the cloud router.Note: Make sure the cloud router is in the same region as the sub-networks it is connecting to.|
+|Google ASN|`65000`|The Autonomous System Number assigned to the cloud router. Use any unused private ASN (64512 - 65534, 4200000000 – 4294967294).|
+
+![alt_text](Image_8.PNG)
+
+In Google Cloud Platform Console, select **Networking** > **Interconnect** > **[VPN](https://console.cloud.google.com/interconnect/vpn)** >> **CREATE VPN CONNECTION**.
+
+**Google Compute Engine VPN gateway**
```

Comment:
Remove: **Google Compute Engine VPN gateway**
Add Step 2: Enter the parameters as shown in the following table for the Google Compute Engine VPN gateway: